phpunit.xml.dist

```diff
@@ -2,15 +2,10 @@
 
 <phpunit
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.3/phpunit.xsd"
-         backupGlobals="false"
-         backupStaticAttributes="false"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.3/phpunit.xsd"
+         beStrictAboutOutputDuringTests="true"
+         beStrictAboutChangesToGlobalState="true"
```

Comment:
Am I correct that this option was introduced in 4.6 (see: [blog post](https://thephp.cc/news/2015/04/phpunit-4-6-and-global-state)), even though it was never [documented](https://phpunit.de/manual/4.6/en/appendixes.configuration.html)? I'm not sure about `beStrictAboutOutputDuringTests`, but there are some online references to that dating back to [2015](https://www.drupal.org/project/drupal/issues/2105583#comment-8966483) so I assume that's fine.

Just wanted to confirm that these will apply to all PHPUnit versions we use for testing.

Reply:
The XSD option was added in https://github.com/sebastianbergmann/phpunit/commit/9101f51cad93f97bea7b277007127635653eadbf#diff-365e6053dce410a51198fc699549f0ae, which was first included in 4.0.0. The comments in the code indicate that the functionality itself was added in 3.8.0, so we should be safe here.

The reason this was added is that I regenerated a new configuration file using `phpunit --generate-configuration` to account for changes in the default configuration.